### PR TITLE
Migration FAQs: added the FAQ links of the migration related tools

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)

added the FAQ links of the migration-related tools in the Migration FAQs document

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6663
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by en-jin19** [comment]
I have finished self-check, PTAL, thanks! @TomShawn

File: faq/migration-tidb-faq.md [hunk 1]
```diff
@@ -5,7 +5,15 @@ summary: Learn about the FAQs related to data migration.
 
 # Migration FAQs
 
-This document summarizes the FAQs related to TiDB data migration.
+This document summarizes the frequently asked questions (FAQs) related to TiDB data migration. 
+
+For the frequently asked questions about migration-related tools, click the corresponding links in the list below:
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:10]:
```suggestion
For the frequently asked questions about migration-related tools, click the following links in the list below:
```

File: faq/migration-tidb-faq.md [hunk 2]
```diff
@@ -5,7 +5,15 @@ summary: Learn about the FAQs related to data migration.
...
+
+- [Backup & Restore FAQ](/br/backup-and-restore-faq.md)
+- [TiDB Binlog FAQ](/tidb-binlog/tidb-binlog-faq.md)
+- [TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md)
+- [Migration FAQs](https://docs.pingcap.com/tidb/stable/migration-tidb-faq)
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:15]:
```suggestion
- [TiDB Migration (DM) FAQs](https://docs.pingcap.com/tidb/stable/migration-tidb-faq)
```

Comment (TomShawn) [inline faq/migration-tidb-faq.md:15]:
```suggestion
- [TiDB Data Migration (DM) FAQs](https://docs.pingcap.com/tidb-data-migration/stable/faq)
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**